content/guides/models/app/settings-page/billing-settings.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Manage your organization's billing settings
+menu:
+  default:
+    parent: settings
+title: Manage billing settings
+weight: 20
+---
+
+Navigate to your user profile page and select your user icon on the top right corner. From the dropdown, choose **Settings**, then click **Billing**.
+
+## Plan details
+
+### Teams plan
+The **Teams plan** section summarizes your organization's current plan, including a summary of monthly charges, as well as your current monthly usage:
+
+- Monthly charges.
+- Seats used. For details and a list of users, click **Manage users**.
+- Tracked hours used. For details, click **View usage**.
+- Amount of monthly data ingested by Weave. For details, click **View usage**.
+
+From here, you can compare plans or talk to Sales.
+
+### Storage
```

Comment:
We also have this doc on managing storage: https://docs.wandb.ai/guides/models/app/settings-page/storage/

There is also this short section:
https://docs.wandb.ai/guides/models/app/settings-page/team-settings/#storage

In the future, it might be worthwhile condensing these and/or referring to one for more info, etc.